Makefile

```diff
@@ -40,16 +44,15 @@ check-reqs:
 shellcheck:
 	$(ROOT_DIR)/tools/shellcheck -e SC1091 jenkins-support *.sh
 
-build:
+build: check-reqs
 	$(call build_image,linux)
 
-list: check-reqs
-	@make --silent build BAKE_ADDITIONAL_FLAGS=--print | jq -r '.target | keys[]'
-
 build-%: check-reqs
-	@$(call check_image,$*)
```

Comment:
I assume you were trying to validate a target is valid here, but groups are not exposed in that  so it prevents from building i.e. all arm images

any thoughts?

if we remove this here should probably also remove it from the test target so that all arm images can be tested and not just individual tests?

Reply:
@dduportal thoughts?

Reply:
yes, that is the intent. I was thinking that "arch" was an implicit thing with both the QEMU or multi buildx workers case.
Let me try to play with this later this morning